webaccess/src/webaccess.cpp

```diff
@@ -2081,6 +2089,66 @@ QString WebAccess::getChildrenHTML(VCWidget *frame, int pagesNum, int currentPag
     return unifiedHTML;
 }
 
+void WebAccess::slotGrandMasterValueChanged(uchar value)
+{
+    GrandMaster::ValueMode value_mode = m_vc->properties().grandMasterValueMode();
+    QString g_value_show;
+    if (value_mode == GrandMaster::Limit)
+    {
+        g_value_show = QString("%1").arg(value, 3, 10, QChar('0'));
+    }
+    else
+    {
+        int p = floor(((double(value) / double(UCHAR_MAX)) * double(100)) + 0.5);
```

Comment:
Perhaps `<cmath>` include is not needed if you use `qFloor`

Reply:
Pushed fix.